dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -91,6 +91,7 @@ COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor"]
 COPY ["etc/rsyslog.conf", "/etc/rsyslog.conf"]
 
+RUN mkdir -p /host/storagemon
```

Comment:
Having a `/host` directory inside the container doesn't make much sense to me, can you pick a different directory? `/usr/share/storagemon`, perhaps?

Reply:
Oh wait did you mean bind /host/pmon/storagemon on the host to /usr/share/storagemon in the container? If so, yes, agreed -- will make that change.

Reply:
Yes, I meant within the container.

Reply:
Done in latest commit. @saiarcot895